### PR TITLE
Story block styling

### PR DESCRIPTION
**Pull request by ToniTepes**
Story block text styling

- pulled in roboto font 
- added prop to story block for big text
- vertically aligned text

solves #26

### CODE REVIEW

File: src/components/bootstrap-overrides.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
```
Comment (easherma) [inline src/components/bootstrap-overrides.css]: groovy!

File: src/components/bootstrap-overrides.css [hunk 2]
```diff
@@ -82,3 +85,16 @@
   line-height: 16px;
   color: #ffffff;
 }
+
+.story-block-text {
+font-family: Roboto, Helvetica, sans-serif;
```
Comment (easherma) [inline src/components/bootstrap-overrides.css]: so in this base, if we're setting a font-family, we should be able to just set that for the entire document and get rid of the commented out font-family declarations above as well. i think its fine for now

File: src/components/story-block.js [hunk 3]
```diff
@@ -12,20 +12,23 @@ const StoryBlock = ({
...
-        <Col sm={6}>
-          {description}
-          <ReactMarkdown source={fullText} />
-        </Col>
-      </Row>
-    </Container>
-  </section>
-)
+    <section>
+      <Container className={bigText ? "story-block-text story-block-big-text" : "story-block-text"} >
```
Comment (easherma) [inline src/components/story-block.js]: this is cool! and I see how that matches with the css classes you declared. However, I'd rather have this be false by default.

File: src/components/bootstrap-overrides.css [hunk 4]
```diff
@@ -82,3 +85,16 @@
...
+font-style: normal;
+font-weight: bold;
```
Comment (easherma) [inline src/components/bootstrap-overrides.css]: by default I don't think it makes sense to have a heavier font weight. this is may be appropriate for the 'big text' sections, but not for the our vision section.

File: src/components/bootstrap-overrides.css [hunk 5]
```diff
@@ -82,3 +85,16 @@
...
+/* or 133% */
+
+color: #707070;
```
Comment (easherma) [inline src/components/bootstrap-overrides.css]: similarly, we want this color on the big-text section but _not_ on the other sections, at least according to the mock

**Review by easherma — CHANGES_REQUESTED** [review]
Looking good. Couple comments, and some changes requested: 

* Some of the CSS you have declared in story-block-text should be in story-block-big-text
* we need to set the prop you created in order to use it. I think its more intuitive to flip it around the other way; have bigText be False by default, and only called true on the story-blocks section

File: src/components/bootstrap-overrides.css (hunk 1 above)
Comment (ToniTepes) [reply src/components/bootstrap-overrides.css]: 👍

File: src/components/bootstrap-overrides.css (hunk 4 above)
Comment (ToniTepes) [reply src/components/bootstrap-overrides.css]: Removed it.

File: src/components/bootstrap-overrides.css (hunk 2 above)
Comment (ToniTepes) [reply src/components/bootstrap-overrides.css]: Removed them

File: src/components/story-block.js (hunk 3 above)
Comment (ToniTepes) [reply src/components/story-block.js]: Got them set to false by default

File: src/components/bootstrap-overrides.css (hunk 4 above)
Comment (ToniTepes) [reply src/components/bootstrap-overrides.css]: I removed bold, but I tried setting the weight lower to no avail.

**Pull request merged**